src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -29,6 +43,35 @@ const UserStatus = ({ profile, setProfile, view }) => {
     setProfile(_.cloneDeep(profile));
   }
 
+  const handleAddressSelect = () => {
+    const addrObj = autocomplete.getPlace();
+    const addr = addrObj.formatted_address;
+
+    if (addr) {
+      profile["address"] = addr;
+      setProfile(_.cloneDeep(profile));
+    }
+  };
+
+  const handleScriptLoad = () => {
+    const options = {
+      type: ["(cities)"],
+    };
+    const addrElem = document.getElementById("address");
+
+    if (window.google !== undefined) {
```

Comment:
If we're only checking for `!== undefined` this can we just check for anything truthy?

`if (window.google)` ?

Reply:
Addressed in 07800cc